samza-api/src/main/java/org/apache/samza/table/retry/TableRetryPolicy.java

```diff
@@ -256,4 +266,30 @@ public BackoffType getBackoffType() {
   public RetryPredicate getRetryPredicate() {
     return retryPredicate;
   }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public Map<String, String> toConfig(Config jobConfig, Config tableConfig) {
+    final Gson gson = new GsonBuilder().excludeFieldsWithModifiers(Modifier.TRANSIENT, Modifier.STATIC).create();
```

Comment:
Is TableRetryPolicy the only one need to override the toConfig function? 
How about read/write function?
Should this implementation be put in somewhere that can be shared?

Reply:
Maybe we could put it in a class in `package org.apache.samza.util`?

Reply:
Yeah it's probably not a bad idea.